app/views/projects/_form.html.erb

```diff
@@ -12,9 +12,12 @@
                             label: :form_projects_title.t + ":",
                             data: { autofocus: true }) %>
 
-  <%= text_area_with_label(form: f, field: :summary, rows: 20,
+  <%= text_area_with_label(form: f, field: :summary, rows: 5,
                            label: :SUMMARY.t + ":") %>
 
   <%= render(partial: "shared/textilize_help") %>
 
+  <%= text_field_with_label(form: f, field: :where,
+                            label: :form_projects_where.t + ":") %>
+
```

Comment:
Just a heads up that there will be a new, easy way to declare an autocomplete field coming up with the Autocompleter refactor PR I hope to merge tonight. 

 I don't know if you would want that for this field, but if you do, the syntax is simple.  You just add `data: { autocompleter: :location }` to the HTML options hash for the `text_field`.

Reply:
Nice!  Adding an autocomplete is on my to do list.  I'll check it out.

Reply:
Adopted!